provisioning_client/tests/hsm_client_riot_ut/hsm_client_riot_ut.c

```diff
@@ -75,15 +77,18 @@ static const char* TEST_CN_VALUE = "riot-device-cert";
 
 static int umocktypes_copy_RIOT_ECC_PRIVATE(RIOT_ECC_PRIVATE* dest, const RIOT_ECC_PRIVATE* src)
 {
-    int result;
-    if (src == NULL)
+    int result = 1;
```

Comment:
We should avoid "guessing" the return value at the beginning. This should be only a declaration and the value should be explicitly set at some branch of the function.

I also think there isn't a return value here that isn't 0. Couldn't you just `return 0`?

Reply:
True there is no reason to fail a umock copy in this UT, seeing as how umock are_equal handles the equivalency check prior to copying, and in this instance we do no memory functions that could fail so if we hit this block we are always successful.

Reply:
Fixed just declaring result